numpy/ma/core.py

```diff
@@ -602,8 +602,10 @@ def filled(a, fill_value=None):
     ----------
     a : MaskedArray or array_like
         An input object.
-    fill_value : scalar, optional
-        Filling value. Default is None.
+    fill_value : Filling value, optional. Can be
```

Comment:
First line should be an approximation of the type, not a description. `array_like, optional` would probably be correct.